testing/internal-integ-testing/src/main/groovy/org/gradle/test/fixtures/server/http/HttpServer.groovy

```diff
@@ -791,11 +765,6 @@ class HttpServer extends ServerWithExpectations implements HttpServerFixture {
             }
 
             void handle(HttpServletRequest request, HttpServletResponse response) {
-
-                if (authenticationScheme.handler.containsUnexpectedAuthentication(request)) {
```

Comment:
❓ This is a safeguard. Why is it removed?

Reply:
as above too broad scope sorry will revisit with proper tooling.